spec.bs

```diff
@@ -38,6 +38,11 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     text: height; url: embedded-content-other.html#attr-dim-height
 spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
   type: dfn
+    text: non-subresource requests; url: #non-subresource-request
+    text: navigation requests; url: #navigation-request
```

Comment:
already exported https://github.com/whatwg/fetch/blob/main/fetch.bs#L1896 (please check to see if there are any others like this here too)